new-lamassu-admin/src/pages/CoinATMRadar/CoinATMRadar.js

```diff
@@ -0,0 +1,180 @@
+import { makeStyles } from '@material-ui/core/styles'
+import React, { useState, memo } from 'react'
+import { useQuery, useMutation } from '@apollo/react-hooks'
+import { gql } from 'apollo-boost'
+
+import { BooleanPropertiesTable } from 'src/components/booleanPropertiesTable'
+import { H4, P, Label2 } from 'src/components/typography'
+import { Button } from 'src/components/buttons'
+import Popper from 'src/components/Popper'
+import { Switch } from 'src/components/inputs'
+import { ReactComponent as HelpIcon } from 'src/styling/icons/action/help/zodiac.svg'
+
+import { mainStyles } from './CoinATMRadar.styles'
+
+const initialValues = {
+  active: false,
+  location: null,
+  commissions: false,
+  supportedCryptocurrencies: false,
+  supportedFiat: false,
+  supportedBuySellDirection: false,
+  limitsAndVerification: false,
+  operatorName: null,
+  operatorPhoneNumber: null,
+  operatorEmail: null
```

Comment:
I though about showing only what had values, and not showing the ones with null value. But I think that this can be better. Maybe using the same structure that you used on the locales screen, and just not adding the properties that will not be used right now.